lnbits/wallets/alby.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+from typing import AsyncGenerator, Optional
+
+import httpx
+from loguru import logger
+
+from lnbits.settings import settings
+
+from .base import (
+    InvoiceResponse,
+    PaymentResponse,
+    PaymentStatus,
+    StatusResponse,
+    Unsupported,
+    Wallet,
+)
+
+
+class AlbyWallet(Wallet):
+    """https://guides.getalby.com/alby-wallet-api/reference/api-reference"""
+
+    def __init__(self):
+        endpoint = settings.alby_api_endpoint
+
+        if not endpoint or not settings.alby_api_key:
+            raise Exception("cannot initialize getalby")
+
+        self.endpoint = endpoint[:-1] if endpoint.endswith("/") else endpoint
+        self.auth = {"Authorization": "Bearer " + settings.alby_api_key}
```

Comment:
If possible it would be useful to pass a user agent header like `lnbits/1.0` so that we can get an estimate of lnbits usage (and what version of the lnbits alby funding source) for anonymous / public statistics

Reply:
done